docs/install_guides/installing-openvino-yocto.md

```diff
@@ -1,40 +1,52 @@
 # Create a Yocto Image with Intel® Distribution of OpenVINO™ toolkit {#openvino_docs_install_guides_installing_openvino_yocto}
+
 This document provides instructions for creating a Yocto image with Intel® Distribution of OpenVINO™ toolkit.
 
 ## System Requirements
-Use the [Yocto Project official documentation](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html#compatible-linux-distribution) to set up and configure your host machine to be compatible with BitBake.
+
+Follow the [Yocto Project official documentation](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html#compatible-linux-distribution) to set up and configure your host machine to be compatible with BitBake.
 
 ## Step 1: Set Up Environment
 
-### Set Up Git Repositories
-The following Git repositories are required to build a Yocto image:
+### 1. Clone the repository.
```

Comment:
How about using plain steps instead of ###headings?

Reply:
Yeah, seems like a better option here :) Will amend these. Thanks.